app/frontend/test/unit/specs/aquifers/components/Search.spec.js

```diff
@@ -12,97 +12,125 @@
     limitations under the License.
 */
 
-import { shallowMount, createLocalVue } from '@vue/test-utils'
-import SearchComponent from '@/aquifers/components/Search.vue'
-import Vuex from 'vuex'
-import axios from 'axios'
-import VueRouter from 'vue-router'
-import auth from '@/common/store/auth.js'
+import { shallowMount, createLocalVue } from "@vue/test-utils"
+import SearchComponent from "@/aquifers/components/Search.vue"
+import Vuex from "vuex"
+import axios from "axios"
+import VueRouter from "vue-router"
+import auth from "@/common/store/auth.js"
```

Comment:
the JS linting is set up with "standardjs" rules which will spit out warnings on double quotes (Derek already commented on a couple other things it will complain about).  This is for JS only (js files or between <script> tags in .vue files)

Reply:
fixed